test/server/cli/test.test.js

```diff
@@ -0,0 +1,56 @@
+import { expect } from 'chai';
+import { runner } from '../../../src/server/cli/test.js';
+
+describe('runner.getOptions()', () => {
+
+	it('should create default options correctly', async() => {
+		const opts = await runner.getOptions();
+		expect(opts).to.deep.include({
+			argv: ['--group', 'test'],
+			readFileConfig: false,
+		});
+		expect(opts.config).to.deep.include({ nodeResolve: true });
+		expect(opts.config).to.have.property('testRunnerHtml').that.is.a('function');
+		expect(opts.config.groups).to.be.an('array').with.length(1);
+		expect(opts.config.groups[0]).to.deep.include({
+			name: 'test',
+			files: ['./test/**/*.test.js']
+		});
+	});
+
+	it('should use --group for the default option', async() => {
+		const opts = await runner.getOptions(['group-name']);
+		expect(opts.argv).to.deep.equal(['--group', 'group-name']);
+	});
+
+	it('should not forward disallowed or stolen options', async() => {
+		const disallowed = ['--browsers', '--playwright', '--puppeteer', '--groups'];
+		const stolen = ['--config'];
```

Comment:
😆 stolen

Reply:
Hah. I tried to avoid it at first but then when I got to `borrowed` it seemed like the most accurate word.